src/main/java/by/vadzimmatsiushonak/bank/api/controller/UserController.java

```diff
@@ -22,17 +32,27 @@
 public class UserController {
 
     private final UserService userService;
+    private final UserMapper userMapper;
 
-    @ApiOperation("Get User")
+    @ApiOperation("Get User with property relations")
     @GetMapping("/{id}")
-    public ResponseEntity<User> findById(@PathVariable Long id) {
-        return ResponseEntity.status(OK).body(userService.findById(id).orElse(null));
+    public ResponseEntity<UserDtoRelations> findById(@PathVariable Long id) {
+        return ResponseEntity.status(OK)
+            .body(userMapper.toDtoRelations(userService.findById(id).orElse(null)));
     }
 
-    @ApiOperation("Get List of Users")
+    @ApiOperation("Get List of Users with property relations")
     @GetMapping
-    public ResponseEntity<List<User>> findAll() {
-        return ResponseEntity.status(OK).body(userService.findAll());
+    public ResponseEntity<List<UserDtoRelations>> findAll() {
+        return ResponseEntity.status(OK)
+            .body(userMapper.toListDtoRelations(userService.findAll()));
     }
 
+    @ApiOperation("Add the User to the Api database")
+    @ResponseStatus(CREATED)
+    @PostMapping
+    public ResponseEntity<UserDto> create(@Valid @RequestBody UserDataRequestDto userRequestDto) {
```

Comment:
We don't need create method here because we can only create Users by using /register endpoint